ansible/playbooks/callback_plugins/custom_retry_runner.py

```diff
@@ -21,6 +21,9 @@ def __init__(self):
         self._last_task_banner = None
         super(CallbackModule, self).__init__()
 
+    def v2_runner_on_start(self, host, task):
```

Comment:
what is the reason for this empty method?

Reply:
It's to get rid of this warning (in pink text): [WARNING]: Failure using method (v2_runner_on_start) in callback plugin
(<display_callback.module.AWXDefaultCallbackModule object at 0x110b72f98>):
'show_per_host_start'